### ISSUE TITLE
Doesn't run on Flutter 2.0

### ISSUE THREAD

**Issue by DennisLit** [espresso3389/flutter_oss_licenses#2]
When I try to run `flutter pub run flutter_oss_licenses:generate.dart` I get the following error

```
type 'MappedIterable<dynamic, dynamic>' is not a subtype of type 'Iterable<String>' in type cast: #0 
```
```
generateLicenseFile (file:///`omitted`/.pub-cache/hosted/pub.dartlang.org/flutter_oss_licenses-1.0.0/bin/generate.dart:75:37)
```

From flutter doctor
`[✓] Flutter (Channel stable, 2.0.1, on Mac OS X 10.15.7 19H524 darwin-x64,locale en-UA)`

**Comment by volmus** [comment]
Same here =/

**Comment by espresso3389** [comment]
Fixed on [1.0.1](https://pub.dev/packages/flutter_oss_licenses/versions/1.0.1).
